flypg/launcher.go

```diff
@@ -304,6 +304,7 @@ func (l *Launcher) getPostgresConfig(config *CreateClusterInput) *api.MachineCon
 	// Metadata
 	machineConfig.Metadata = map[string]string{
 		"managed-by-fly-deploy": "true",
+		"fly_platform_version":  "v2",
```

Comment:
what about doing it like  https://github.com/superfly/flyctl/blob/ae564072a863e385c112042b997d8dce25bf4168/internal/command/deploy/machines.go#L618

Reply:
in fact, that function has this also https://github.com/superfly/flyctl/blob/ae564072a863e385c112042b997d8dce25bf4168/internal/command/deploy/machines.go#L624